doc/shared/sunmemory/SUNMemory_CUDA.rst

```diff
@@ -68,10 +68,10 @@ The implementation provides the following operations defined by the
 
    **Returns:**
 
-   * An ``int`` flag indicating success (zero) or failure (non-zero).
+   * A :c:type:`SUNErrCode` indicating success or failure
```

Comment:
```suggestion
   * A :c:type:`SUNErrCode` indicating success or failure.
```

Reply:
This function returns a `SUNMemory` object